api/seismic_site/settings.py

```diff
@@ -46,6 +46,7 @@ class Base(Configuration):
         "rest_framework",
         "storages",
         "taggit",
+        "corsheaders",
```

Comment:
Incomplete setup for the `django-cors-header` app. Please refer to [the docs](https://github.com/adamchainz/django-cors-headers#setup). Hint: the middleware is missing and it must be as high as it can be in the list (before `whitenoise.middleware.WhiteNoiseMiddleware`)

Reply:
Sure!